Jenkinsfile_CNP

```diff
@@ -60,15 +64,38 @@ withNightlyPipeline(type, product, component) {
 
   env.envName = params.EnvironmentChoice
   env.envDriver = params.Browser
+  env.runJurorTransformation = params.runJurorTransformation
+  env.runJurorTransformationMulti = params.runJurorTransformationMulti
+  env.runJurorTransformationWIP = params.runJurorTransformationWIP
+  env.runShakedown = params.runShakedown
+  
 
   afterAlways('fullFunctionalTest') {
     publishHTML target: [
       allowMissing         : true,
       alwaysLinkToLastBuild: true,
       keepAll              : true,
-      reportDir            : "target/TestRunner_SmokeTest/",
+      reportDir            : "target/TestRunner_/JurorTransformation",
       reportFiles          : "cucumber.html",
-      reportName           : "Cucumber Smoke Test Report"
+      reportName           : "Cucumber JurorTransformation Test Report"
+    ]
+
+    publishHTML target: [
+      allowMissing         : true,
+      alwaysLinkToLastBuild: true,
+      keepAll              : true,
+      reportDir            : "target/TestRunner_JurorTransformationMulti/",
+      reportFiles          : "cucumber.html",
+      reportName           : "Cucumber JurorTransformationMulti Test Report"
+    ]
+
+    publishHTML target: [
+      allowMissing         : true,
+      alwaysLinkToLastBuild: true,
+      keepAll              : true,
+      reportDir            : "target/TestRunner_JurorTransformationWIP/",
```

Comment:
you'll need to export the html in each runner for example in smoke test it required `"html:target/TestRunner_SmokeTest/cucumber.html"`